spec/requests/tool_integration/decision_trees_spec.rb

```diff
@@ -1,7 +1,9 @@
 RSpec.describe 'Decision Trees', type: :request, features: [:health_check, :registration] do
   %W(
-    /en/tools/#{ToolMountPoint::DecisionTrees::EN_ID}
-    /cy/tools/#{ToolMountPoint::DecisionTrees::CY_ID}
+    /en/tools/health-check-questions
+    /cy/tools/health-check-questions
+    /en/tools/workplace-pension-advice-tool
+    /cy/tools/workplace-pension-advice-tool
```

Comment:
the current welsh url of the tool is found at https://www.moneyadviceservice.org.uk/cy/tools/cyfrifiannell-cyfraniadau-pensiwn-gweithle